pkg/serializer/internal/metrics/origin_mapping.go

```diff
@@ -12,14 +12,18 @@ import (
 func metricSourceToOriginProduct(ms metrics.MetricSource) int32 {
 	const serieMetadataOriginOriginProductAgentType = 10
 	const serieMetadataOriginOriginProductDatadogExporterType = 19
+	const serieMetadataOriginOriginProductGPU = 38 // ref: https://github.com/DataDog/dd-source/blob/main/domains/metrics/shared/libs/proto/origin/origin.proto#L277
```

Comment:
```suggestion
	const serieMetadataOriginOriginProductGPU = 38 // ref: https://github.com/DataDog/dd-source/blob/276882b71d84785ec89c31973046ab66d5a01807/domains/metrics/shared/libs/proto/origin/origin.proto#L277
```
Should be a permalink IMO